Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -616,6 +625,10 @@ extension TelemetryWrapper {
             GleanMetrics.Tabs.openTabTray.record()
         case (.action, .close, .tabTray, _, _):
             GleanMetrics.Tabs.closeTabTray.record()
+        case (.action, .press, .tabToolbar, .tabView, _):
+            GleanMetrics.Tabs.pressTabToolbar.record()
+        case (.action, .press, .tab, _, _):
+            GleanMetrics.Tabs.pressTab.record()
```

Comment:
```suggestion
            GleanMetrics.Tabs.pressTopTab.record()
```

Reply:
Yes @nbhasin2, good notice, I will make these changes, thank you!!